<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTcpClient</class>
 <widget class="QMainWindow" name="MyTcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTcpClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>网络设置</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>服务器IP地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtIP">
              <property name="text">
               <string>192.168.100.102</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>服务器端口号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="edtPort">
              <property name="text">
               <string>5678</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>序列号设置</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="serno_0">
              <property name="text">
               <string>123456789</string>
              </property>
              <property name="maxLength">
               <number>9</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="serno_1">
              <property name="text">
               <string>1111111</string>
              </property>
              <property name="maxLength">
               <number>9</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="serno_2">
              <property name="text">
               <string>222222</string>
              </property>
              <property name="maxLength">
               <number>9</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="serno_3">
              <property name="text">
               <string>3333333</string>
              </property>
              <property name="maxLength">
               <number>9</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmd_serno">
              <property name="text">
               <string>设置序列号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>清除序列号</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>326</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="action_import_serno"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="action_cfg_deadline"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_import_serno">
   <property name="text">
    <string>载入序列号</string>
   </property>
  </action>
  <action name="action_cfg_deadline">
   <property name="text">
    <string>续期</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
